packages/client/src/components/app/Repeater.svelte

```diff
@@ -22,7 +23,10 @@
     <Placeholder />
   {:else if rows.length > 0}
     {#each rows as row, index}
-      <Provider data={{ ...row, index }} scope={ContextScopes.Local}>
+      <Provider
+        data={{ ...row, index }}
+        scope={nested ? ContextScopes.Global : ContextScopes.Local}
```

Comment:
Is there any reason this can't always just be global?

Reply:
Ah that makes sense - we would need a new way to key the repeater rows in the global context. Is `global` a better name for this property then?

Reply:
Yep agreed. It does step on the current nested flag used elsewhere in the block components. I've just replaced it with a configurable scope prop instead.